tests/integration_test/services_test.py

```
@@ -23655,7 +23710,7 @@
     assert actual["error_code"] == 0
     assert actual["success"]
     assert len(actual["data"]["logs"]) == 10
-    assert actual["data"]["total"] == 11
+    assert actual["data"]["total"] == 10
```

Ruff (0.12.2)

Use of `assert` detected

(S101)